specify/specify7
# Hide CollectionObjectType on the frontend

Fixes #5222

This could've been added to #5191 but having a separate PR for this should make it super easy to revert these changes when we release 7.9.8

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions



- Go to Data Entry > Click on the pencil icon > Select use Custom Settings
- [ ] Verify Collection Object Type is hidden in the list of possible tables
- Go to Workbench > Create New 
- [ ] Verify Collection Object Type is hidden in the list of base tables
- Go to Schema Config
  - [ ] Verify Collection Object Type is hidden in the list of base tables
  - Select Collection Object as base table
  - [ ] Verify collectionObjectType is hidden under relationships 
- Go App Resources > Form Definition > Select a `.views` resource
- [ ] Verify Collection Object Type is hidden in the list of tables

#### Verify CoType is hidden in relation mapper
- Go to Query Builder
  - [ ] Verify CollectionObjectType is not visible in the mapper for a Collection query
  - [ ] Verify CollectionObjectType is not visible in the mapper for a CollectionObject query
- Go to Workbench
  - [ ] Verify CollectionObjectType is not visible in the mapper when using Collection as base table
  - [ ] Verify CollectionObjectType is not visible in the mapper when using CollectionObject as base table
- Go to Schema Config > CollectionObject
  - Click on Uniqueness Rules > Add Uniqueness Rule > Click on the pencil icon for the new rule
  - [ ] Verify CollectionObjectType is not visible
